examples/docking-antibody-antigen/docking-antibody-antigen-CDR-NMR-CSP-full.cfg

```diff
@@ -62,7 +62,7 @@ reference_fname = "data/4G6M-matched.pdb"
 [clustfcc]
 
 [seletopclusts]
-top_cluster = []
```

Comment:
So this is no longer needed? The default being then to select all clusters?

Reply:
precisely! the default value for `top_cluster` is 1000, so that if the number of cluster is lower (as it typically is) we select all of them